runtime/oti/j9nonbuilder.h

```diff
@@ -5834,11 +5836,6 @@ typedef struct J9JavaVM {
 	struct J9HashTable* ensureHashedClasses;
 #if JAVA_SPEC_VERSION >= 19
 	U_64 nextTID;
-	j9object_t *liveVirtualThreadList;
-	omrthread_monitor_t liveVirtualThreadListMutex;
-	volatile BOOLEAN inspectingLiveVirtualThreadList;
-	UDATA virtualThreadLinkNextOffset;
```

Comment:
I don't see an update to DDR code to deal with the removal of `virtualThreadLinkNextOffset`; as a result, the `!vthreads` command is now broken: #17084 tracks that problem.